apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/LoadingIndicator/index.tsx

```diff
@@ -3,7 +3,7 @@ import { Box, InfiniteLoader } from "@ledgerhq/native-ui";
 
 export default function LoadingIndicator() {
   return (
-    <Box height={40} pt={4} accessible accessibilityRole="progressbar">
```

Comment:
Oops, I didn't mean to do it here. 
But if `LoadingIndicator` is shared, we shouldn't apply component-specific styles directly to it. Since I use it in the horizontal list, I need to restyle it within the Horizontal List component. What do you think ?